libraries/AP_HAL_ChibiOS/hwdef/Morakot/defaults.parm

```diff
@@ -0,0 +1,97 @@
+# Define ethernet settings
+NET_DHCP 0
+NET_ENABLE 1
+NET_GWADDR0 192
+NET_GWADDR1 168
+NET_GWADDR2 1
+NET_GWADDR3 1
+NET_IPADDR0 192
+NET_IPADDR1 168
+NET_IPADDR2 1
+NET_IPADDR3 21
+NET_NETMASK,24
+NET_OPTIONS,0
+NET_P1_IP0,255
+NET_P1_IP1,255
+NET_P1_IP2,255
+NET_P1_IP3,255
+NET_P1_PORT,14550
+NET_P1_PROTOCOL,2
+NET_P1_TYPE,1
+NET_P2_TYPE,0
+NET_P3_TYPE,0
+NET_P4_TYPE,0
+
+
+NTF_BUZZ_ON_LVL,1
+NTF_BUZZ_PIN,-1
+NTF_BUZZ_TYPES,5
+NTF_BUZZ_VOLUME,100
+NTF_DISPLAY_TYPE,0
+NTF_LED_BRIGHT,3
+NTF_LED_LEN,1
+NTF_LED_OVERRIDE,0
+NTF_LED_TYPES,123079
+
+# Serial0 setting (USB/OTG)
+SERIAL0_BAUD,115
+SERIAL0_PROTOCOL,2
+
+# Serial1 setting (DJI / USART1)

```

Comment:
remove section...only  add a define DEFAULT_SERIAL2_PROTOCOL SerialProtocol_MSP_DisplayPort line in hwdef